tests/test_arcaflow_plugin_stressng.py

```diff
@@ -61,18 +61,17 @@ def test_functional_cpu(self):
                 print(e)
 
         self.assertEqual(yaml.safe_load(result), reference)
```

Comment:
The code here is going to catch and "eat" the YAML loading error (yes, it prints it out, but who's really going to notice??), and we're _hoping_ that the assertion will fail as a result of the error.  I realize that you've made no changes here, but this is a very weak assumption, whereas it would be very straightforward to simply let the exception propagate out to the test framework (by removing the `try` block or by calling `raise` after the `print()`) or to explicitly call `unittest.fail()` in the `except` clause.